### DISCUSSION TITLE
Draw Order

### DISCUSSION THREAD

**Discussion by Ron5s** [Kode/Kha#1287] [General]
Is there a way to change draw order other than writing them in order from front to back?

Thanks for the help.

**Comment by RobDangerous** [comment]
What makes you think there is a draw order?

**Comment by Ron5s** [comment]
Oh! interesting question.  I guess I assumed that there was a draw order because if I run the following code. the fillrects' will be drawn over each other, with the last red fillrect drawn last on top of the previous 2.

graphics.color = Color.fromBytes(0, 255, 0);
graphics.fillRect(32, 32, 128, 128);
graphics.color = Color.fromBytes(0, 0, 255);
graphics.fillRect(32, 32, 128, 128);
graphics.color = Color.fromBytes(255, 0, 0);
graphics.fillRect(32, 32, 128, 128);

Is there a way to draw the green fillrect over the other 2 without rearranging the code?

Thanks for the help.

**Comment by RobDangerous** [comment]
No, there is no draw order in Kha, everything is drawn in the order you draw it in.

**Comment by Ron5s** [comment]
Ok understood.  Thank you for the help.
